week5/sheldon/index.js

```diff
@@ -0,0 +1,128 @@
+// --------------------------------------first question
+// const isString = str => {
+//   if (str === +str) {
+//     return false;
+//   } else {
```

Comment:
you can simplify this conditional statement, you don't need the else

Reply:
changed to return true